modules/intro/pages/index.adoc

```diff
@@ -80,9 +80,9 @@ for the most up-to-date news on pyTigerGraph.
 |===
 
 == Peformance Notes
-pyTigerGraph is a feature rich package that allows you to interact with TigerGraph databases from Python.
-However, it is important to note that there will be a performance overhead when using pyTigerGraph compared to using cURL-like requests directly to the TigerGraph REST++ API.
-This is because pyTigerGraph is a Python package that wraps the REST++ API and adds additional functionality such as URL building, logging, and authentication handling, as well as error checking.
+pyTigerGraph is a feature-rich package that allows you to interact with TigerGraph databases using Python.
+It enhances the REST++ API by adding extra features, such as URL setup, log creation, authentication, and error checks.
+Therefore, pyTigerGraph may perform slower than direct cURL requests to the TigerGraph REST++ API.
```

Comment:
```suggestion
Therefore, pyTigerGraph may perform slower than direct HTTP requests to the TigerGraph REST++ API.
```
Technically:
REST is an architectural style for client-server communication.
HTTP is the most common communication protocol used to implement REST; TigerGraph uses HTTP.
cURL is the most common utility used to send and receive HTTP messages in Linux.  This is the customer's choice. They don't have to use cURL to make use of TigerGraph's RESTful HTTP endpoints.